kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/InboxServiceImpl.java

```diff
@@ -56,7 +56,7 @@ public SubscriptionPackMessage getInboxMessage(Subscription subscription) throws
         int daysIntoWeek = daysIntoPack % DAYS_IN_WEEK;
 
         if (subscription.getStatus() == SubscriptionStatus.COMPLETED) {
-            int totalWeeksInPack = pack.getWeeklyMessages().size() * pack.getMessagesPerWeek();
+            int totalWeeksInPack = pack.getWeeklyMessages().size() / pack.getMessagesPerWeek();
```

Comment:
I was on the fence about this -- I had figured, if we can calculate it, no need to store it. :) But I'm happy to store it if you guys like that approach better.